cli/tests/unit/flock_test.ts

```diff
@@ -1,102 +1,152 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 import { assertEquals, unitTest } from "./test_util.ts";
+import { readAll } from "../../../test_util/std/io/util.ts";
 
 unitTest(
   { perms: { read: true, run: true, hrtime: true } },
   async function flockFileSync() {
-    const path = "cli/tests/testdata/fixture.json";
-    const script = (exclusive: boolean, wait: number) => `
-          const { rid } = Deno.openSync("${path}");
-          Deno.flockSync(rid, ${exclusive ? "true" : "false"});
-          await new Promise(res => setTimeout(res, ${wait}));
-          Deno.funlockSync(rid);
-      `;
-    const run = (e: boolean, w: number) =>
-      Deno.run({ cmd: [Deno.execPath(), "eval", "--unstable", script(e, w)] });
-    const firstBlocksSecond = async (
-      first: boolean,
-      second: boolean,
-    ): Promise<boolean> => {
-      const firstPs = run(first, 1000);
-      await new Promise((res) => setTimeout(res, 250));
-      const start = performance.now();
-      const secondPs = run(second, 0);
-      await secondPs.status();
-      const didBlock = (performance.now() - start) > 500;
```

Comment:
It's not reliable to do stuff like this on the CI. The fix below changes the code to signal and waits when it's at different parts of the code and records the time it enters and exits the locks to check if blocking occurred.